source/includes/api/_dashboard.md

```diff
@@ -233,13 +233,13 @@ In order to perform this operation, the following conditions must be met:
 - the user must be logged in and belong to the same application as the dashboard
 - the user must match one of the following:
   - have role `ADMIN`
-  - have role `MANAGER` and be the dashboard's owner (through the `user-id` field of the dashboard)
+  - have role `MANAGER` or `USER` and be the dashboard's owner (through the `user-id` field of the dashboard)
```

Comment:
https://github.com/resource-watch/resource-watch-manager/blob/35c71d09305c2d778ed49e07f04d46f324f9d937/spec/controllers/api/dashboards_update_spec.rb#L43

This was modified recently to mimic the behavior already found on other similar microservices - USERs cannot edit resources (even their own)